getting-started/git-boot-camp.rst

```diff
@@ -286,11 +286,24 @@ Creating a Pull Request
 You should include the issue number in the title of the PR,
 in the format ``gh-NNNNN: <PR Title>``.
 
-You might also use `special keywords
-<https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword>`_
-to automatically close the linked issue once the PR is merged. However, you
-should avoid them if other PRs need to be merged before the issue can be
-closed (e.g. backports to other branches).
+Linking to Issues and Pull Requests
+-----------------------------------
+
+You can link to issues and pull requests using ``gh-NNNNN`` (this form is
+preferred over ``#NNNNN``).  If the reference appears in a list, the link
```

Comment:
For commits and PR titles sure, but why in the comment body? `#NNNN` is the de-facto standard form on GitHub, and is naturally extended to issues on other repos and orgs than just the current one with `org/repo#NNN`, and references to issues/PRs from elsewhere, whereas I know of no such equivalent for `gh-NNN`.

Reply:
> but why in the comment body

Probably for another potential migration in the future.

Older BPO issues also used hashed issue ids and they already cannot be automatically distinguished from later introduced GitHub PRs. So there's no need to continue such a confusion.

Reply:
Thanks, that makes sense I guess. Unfortunately, even as a triager I can't mark my own review comments as resolved (unless/until python/core-workflow#460 is addressed), yet I _can_ mark others' comments as resolved as a PR author (which doesn't make any sense; if anything it should be the opposite), so feel free to resolve this one.